config/locales/en.yml

```diff
@@ -910,6 +918,10 @@ en:
           attributes:
             plan:
               not_allowed: "not allowed in this context"
+            keys:
+              invalid: Invalid. Use only alphanumeric characters [0-9, a-z, A-Z], hyphen-minus (-), no spaces and 40 characters at most.
+            user_key:
+              invalid: Invalid. Use only alphanumeric characters [0-9, a-z, A-Z], hyphen-minus (-), no spaces and 100 characters at most.
```

Comment:
similar to my other comment, where is `100` coming from? :sweat_smile: 

[here](https://github.com/3scale/porta/blob/b122bced37550ea23661cee1e7a13c4ccd50d105/app/models/cinstance.rb#L122-L123) it says 256